src/dex-ui/pages/dao/DAOProposals/Forms/DAOTokenTransferDetailsForm.tsx

```diff
@@ -119,36 +143,39 @@ export function DAOTokenTransferDetailsForm() {
         isInvalid={Boolean(errors?.recipientAccountId)}
         errorMessage={errors?.recipientAccountId && errors?.recipientAccountId?.message}
       />
-      <FormInput<"tokenId">
-        inputProps={{
-          id: "tokenId",
-          label: "Asset",
-          type: "text",
-          placeholder: "Enter a token ID",
-          register: {
-            ...register("tokenId", {
-              required: { value: true, message: "A token ID is required." },
-            }),
-          },
-          value: state?.tokenId,
-        }}
-        isInvalid={Boolean(errors?.tokenId)}
-        errorMessage={errors?.tokenId && errors?.tokenId?.message}
-      />
-      {/*
-       *  TODO: Replace Assets Input with Dropdown
-       *  <FormDropdown
-       *   label="Assets"
-       *   placeholder="Select an asset"
-       *   data={assetDropdownOptions}
-       *   isInvalid={Boolean(errors?.tokenId)}
-       *   errorMessage={errors?.tokenId && errors?.tokenId?.message}
-       *   register={register("tokenId", {
-       *     required: { value: true, message: "A token is required." },
-       *     onChange: (e: ChangeEvent<HTMLSelectElement>) => setValue("tokenId", e.target.value),
-       *   })}
-       * />
-       */}
+      {daoType === DAOType.MultiSig ? (
+        <Flex direction="column" alignItems="left" gap="1">
+          <FormDropdown
+            label="Assets"
+            placeholder="Select an asset"
+            data={assetDropdownOptions}
+            isInvalid={Boolean(errors?.tokenId)}
+            errorMessage={errors?.tokenId && errors?.tokenId?.message}
+            register={register("tokenId", {
+              required: { value: true, message: "A token is required." },
+              onChange: (e: ChangeEvent<HTMLSelectElement>) => setValue("tokenId", e.target.value),
+            })}
+          />
+          <Text textStyle="h4">Balance:&nbsp;{selectedAsset?.balance ?? 0}</Text>
```

Comment:
Place the Balance under the Amount input field. We will replace the Amount Form Input component with the Token Input component in a future PR but the balance amount should be co-located with the Amount Form Input for now.

![Screen Shot 2023-07-24 at 6 56 42 PM](https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/54907098/e08e9504-f121-4de9-9dfa-3647694bdc6d)
^Token Input component (input component with min button, max button, and token symbol) does not need to be included in this PR:

Reply:
Made the changes are per your suggestions.